lib/bootstrap.js

```diff
@@ -207,7 +207,8 @@ function applyPrefix(data, site) {
 function load(data, site) {
   let promiseComponentsOps, promisePagesOps, promiseUrisOps, promiseListsOps, promiseUsersOps,
     prefix = site.prefix,
-    compData = applyPrefix(data, site);
+    compData = applyPrefix(data, site),
+    locals = { site, components: files.getComponents() };
```

Comment:
I'm wondering if we need to stub some of the other properties available on the object by default. 

- `locals.url`
- `locals.query`
- `locals.params`
- `locals.edit`

https://docs.clayplatform.com/docs/glossary#locals

Those are just some that are nice to not have to check for when writing model.js files